README.md

```diff
@@ -128,51 +130,75 @@ involve IPFS.
 We will be using the following tools and services:
 
 * [Digital Ocean](https://www.digitalocean.com) as the virtual machine provider
-* [Vagrant](https://www.vagrantup.com) to provision the cloud servers
+* [Terraform](https://www.terraform.io) to provision the cloud servers
 
 The following steps assume you have a Digital Ocean account and the above listed software
 installed on your local machine, which can be the same device running OBS Studio.
 
-1. Install vagrant plugins:
+1. Clone this repository and work from the `terraform` directory:
+
+        git clone https://github.com/tomeshnet/ipfs-live-streaming.git
+        cd ipfs-live-streaming/terraform
+
+1. From your domain name registerar, point name servers to Digital Ocean's name servers:
 
-        vagrant plugin install vagrant-digitalocean vagrant-scp
+        ns1.digitalocean.com
+        ns2.digitalocean.com
+        ns3.digitalocean.com
+
+    Then store the domain name in your local environment:
+
+        echo -n YOUR_DOMAIN_NAME > .keys/domain_name
+
+1. Obtain a read-write access token from your Digital Ocean account and store it in your local
+    environment:
+
+        echo -n YOUR_DIGITAL_OCEAN_ACCESS_TOKEN > .keys/do_token
 
 1. Generate RSA keys to access your Digital Ocean VMs:
 
-        ssh-keygen -t rsa -f ~/.ssh/ipfs_live_streaming_rsa
+        ssh-keygen -t rsa -f .keys/id_rsa
+
+    Add the SSH key to your Digital Ocean account, then copy the SSH fingerprint to local
+    environment:
+
+        echo -n YOUR_SSH_FINGERPRINT > .keys/ssh_fingerprint
 
-1. Obtain a read-write access token from your Digital Ocean account and export it in
-    your local environment:
+1. [Install Terraform](https://www.terraform.io/intro/getting-started/install.html), add it to
+    your path, then run initialization from our `terraform` working directory:
 
-        export DO_ACCESS_TOKEN=<YOUR_DIGITAL_OCEAN_ACCESS_TOKEN>
+        terraform init
 
-1. Clone this repository and from the `vagrant` directory, provision the streaming servers
-    by running:
+1. Provision the streaming servers by running:
 
-        vagrant up
+        terraform apply
```

Comment:
Currently you need
```
touch .keys/rtmp_yggdrasil
touch .keys/ipfs_id
```

otherwise complains they are missing

Reply:
Non-ideal but can't think of a better way for now. In `.gitignore`:

```
.keys/*
!.keys/rtmp_yggdrasil
!.keys/ipfs_id
```